### ISSUE TITLE
What is the safest place to put secret keys in the backend

### ISSUE THREAD

**Issue by SumayaG** [AnomalyInnovations/serverless-stack-com#203]
Hi,

I have a SECRET KEY for STRIPE that i have to use in the backend. This secret key should not be exposed to anyone. Where do you think would be the safest place to put this and how to do it. 

Regards

**Comment by jayair** [comment]
@SumayaG Yeah it definitely shouldn't be in your code. You could make it a part of your build service. Or use AWS SSM. Here is some more info on it https://hackernoon.com/you-should-use-ssm-parameter-store-over-lambda-env-variables-5197fc6ea45b

**Comment by SumayaG** [comment]
@jayair Thanks, I'll look into AWS SSM.
